README.md

```diff
@@ -82,6 +82,10 @@ run into trouble feel free to file an issue.
 
 Peer-reviewed publications part of journals, conferences and workshops:
 
+#### 2021
+
+ * [Solving Configuration Problems with ASP and Declarative Domain Specific Heuristics](http://ceur-ws.org/Vol-2945/21-RT-ConfWS21_paper_4.pdf)
```

Comment:
```suggestion
 * [Solving Configuration Problems with ASP and Declarative Domain-Specific Heuristics](http://ceur-ws.org/Vol-2945/21-RT-ConfWS21_paper_4.pdf)
```

Reply:
Hmm, that hyphen did not make it into DBLP...